src/elements/TabbedContainer/TabbedContainer.less

```diff
@@ -0,0 +1,54 @@
+.tabs-wrapper {
```

Comment:
It might be best to name the classes after the component and also give a `ui` namespace - instead of `tabs-wrapper`, `ui-tab-container` is likely more appropriate